### DISCUSSION TITLE
Change Branches

### DISCUSSION THREAD

**Discussion by stickz** [Novik/ruTorrent#2606] [General]
@Novik Could you change the default branch to `v4.2`? And rename `master` to `v4.1`? I don't have this level of access.

**Comment by TrimmingFool** [comment]
@stickz What benefit do you think this will have? I would suggest using the [GitLab flow branch strategy](https://www.gitkraken.com/learn/git/best-practices/git-branch-strategy#gitlab-flow-branch-strategy).
> Versioned Release: each release has an associated release branch that is based off the main branch. Bug fixes should be merged into the main branch first, before being cherry-picked into the release branch.

**Comment by Novik** [comment]
@stickz
https://stackoverflow.com/questions/2862590/how-to-replace-master-branch-in-git-entirely-from-another-branch
https://stackoverflow.com/questions/2763006/make-the-current-git-branch-a-master-branch
etc.

And IMHO suggestion of @TrimmingFool make sense.

**Comment by stickz** [comment]
Okay, I made a new branch for `v4.1`. Then brought master up to date with `v4.2`. Development of `v4.3` is about to start.
